recipe/meta.yaml

```diff
@@ -25,15 +30,21 @@ requirements:
     - ordered-set
     - python-dateutil
     - weakrefmethod >=1.0.3
+    # unvendored six
+    - six 1.15.0
```

Comment:
Six is fully backward compatible. We could use a lower bound instead of a hard pin.

Reply:
ok,

done with `six >=1.16.0`, 

six 1.15.0 has this problem: https://github.com/benjaminp/six/issues/39